bindings_ffi/tests/test_android.kts

```diff
@@ -0,0 +1,27 @@
+import kotlinx.coroutines.*
+import kotlin.system.*
+import uniffi.xmtpv3.*
+
+runBlocking {
+    try {
+        val client = uniffi.xmtpv3.createClient("http://localhost:5556", false)
```

Comment:
There will eventually need to be a envvar override for the url in the future. But thats not a now problem.